tools/ci_build/github/linux/run_python_dockerbuild.sh

```diff
@@ -8,6 +8,10 @@ x) BUILD_EXTR_PAR=${OPTARG};;
 esac
 done
 
+if [ "$BUILD_EXTR_PAR" = "-d AZURE" ]; then
+    export ADDITIONAL_DOCKER_PARAMETER="${ADDITIONAL_DOCKER_PARAMETER} -u root"
```

Comment:
Please do not use root to build any code

Reply:
Agree - totally we should not use root to build.
Problem is without root we failed at "yum install SSL..." which required by triton client.
The best way to solve the problem is to rebuild a docker image with package installed ahead of time, problem is - do we still have time to do that, and how?

Reply:
Have you tried to add it to: https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh ?